code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -234,7 +234,8 @@ namespace PlayFab
             auto a = result.QosServers;
             for (auto it = a.begin(); it != a.end(); ++it)
             {
-                api->regionMap[it->Region] = move(it->ServerUrl);
+                //api->regionMap[it->Region] = move(it->ServerUrl);
+                api->regionMap["it->Region as a std::string"] = move(it->ServerUrl);
```

Comment:
This line should remain unchanged with the latest API spec. I had to emulate it since it will become a string.